Dockerfile

```diff
@@ -1,15 +1,14 @@
 # OS+dist should be kept in sync with .travis.yml
 FROM ubuntu:focal
 
-RUN apt-get update && apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev git netcat
+RUN apt-get update \
+ && apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev git netcat
 RUN curl -L https://raw.githubusercontent.com/yyuu/pyenv-installer/master/bin/pyenv-installer | bash
 
 ENV PYENV_ROOT /root/.pyenv
 ENV PATH $PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
 
 RUN eval "$(pyenv init -)" && eval "$(pyenv virtualenv-init -)"
-RUN pyenv install 3.6.15
-RUN pyenv install 3.7.9
```

Comment:
Please consider adding 3.11 and 3.12 as well. And also probably a more recent pypy3